portfolio.js

```diff
@@ -267,4 +267,36 @@ form.addEventListener('submit', (e) => {
     formButton.reportValidity();
     e.preventDefault();
   }
-});
\ No newline at end of file
+});
+
+// set up local storage
+
+const nameInput = document.getElementById('name');
+const mailInput = document.getElementById('email');
+const messageInput = document.getElementById('message');
+
+const nameValue = nameInput.value;
+const mailValue = mailInput.value;
+const messageValue = messageInput.value;
+
+formButton.onclick = function saveUserData() {
```

Comment:
- [ ] When the user changes the content of any input field in the form the data object should be updated and saved to the local storage, kindly implement this so the correct data is saved